code/parse/sexp.cpp

```diff
@@ -682,6 +682,7 @@ SCP_vector<sexp_oper> Operators = {
 	{ "hud-set-custom-gauge-active",	OP_HUD_SET_CUSTOM_GAUGE_ACTIVE,			2, 	INT_MAX, 	SEXP_ACTION_OPERATOR,	},
 	{ "hud-set-builtin-gauge-active",	OP_HUD_SET_BUILTIN_GAUGE_ACTIVE,		2, 	INT_MAX,	SEXP_ACTION_OPERATOR,	},
 	{ "hud-set-text",					OP_HUD_SET_TEXT,						2,	2,			SEXP_ACTION_OPERATOR,	},	//WMCoolmon
+	{ "hud-set-xstr",                   OP_HUD_SET_XSTR,                        3,  3,          SEXP_ACTION_OPERATOR,   },  //MjnMixael
```

Comment:
Thanks for the quick fix! Just one last thing I was pondering, it looked like all the other `-xstr` related sexps have `-xstr` appended to the end of the original name, so I was just wondering what your thoughts on doing the same here (ie 
having the name be `hud-set-text-xstr`)?

Reply:
I'll be honest, my reasons are entirely selfish... using hud-set-xstr will in-place replace all of BtA's Lua `hud-set-xstr` SEXPs that turned out to be buggy which is why an in-engine solution was preferred.

Reply:
Just as a potential thought another way is to use the deprecation string replace in `get_sexp()` (like `remove-weapons` and `clear-weapons`)

Anyhow, I'm always a sucker for internal consistency such as using `hud-set-text-xstr` but certainly won't hold up the PR over it :D

Reply:
I think there's competing consistencies here then. `sexp-xstr` is one, sure. But all the `hud-set` sexps are `hud-set-type` and Text is distinct from Xstr. For the latter, `hud-set-xstr` is consistent with the other hud sexps.

Reply:
Oh yeah that makes sense, thanks for the extra discussion!